core/src/main/java/io/aiven/klaw/validation/TopicRequestValidatorImpl.java

```diff
@@ -125,6 +125,13 @@ public boolean isValid(
       return false;
     }
 
+    // Check if topic exists on update requests
+    if (RequestOperationType.UPDATE == topicRequestModel.getRequestOperationType()
```

Comment:
What about for delete and claim as well?

I think create is actually the only one where this doesn't count?

Reply:
Added DELETE to the condition. I don't see the need for CLAIM

Reply:
Adding claim, and env checks for other types